### PR TITLE
upgrade to laravel 5.5 and beyond

### PR DESCRIPTION
**Pull request by crissi**

### CODE REVIEW

File: src/A2way/LaravelTenantMigrate/MigrateTenantDabataseCommand.php [hunk 1]
```diff
@@ -37,11 +37,11 @@ public function __construct()
 	 *
 	 * @return mixed
 	 */
-	public function fire()
+	public function handle()
 	{
 		$connectionName = $this->argument('connection-name');
 		$databaseName = $this->argument('database-name');
-		
+		\DB::purge($connectionName);
```
Comment (budhajeewa) [inline src/A2way/LaravelTenantMigrate/MigrateTenantDabataseCommand.php]: @crissi : Why was this necessary? :thinking:

**Comment by crissi** [comment]
It did not work without it, seems others have success with it too: https://stackoverflow.com/questions/31041893/laravel-change-database-connection-run-time

Få Outlook til iOS<https://aka.ms/o0ukef>

________________________________
Fra: Supun Budhajeewa <notifications@github.com>
Sendt: tirsdag, marts 5, 2019 4:27 PM
Til: a2way-com/laravel-tenant-migrate
Cc: Christian; Mention
Emne: Re: [a2way-com/laravel-tenant-migrate] upgrade to laravel 5.5 and beyond (#12)


@budhajeewa commented on this pull request.

________________________________

In src/A2way/LaravelTenantMigrate/MigrateTenantDabataseCommand.php<https://github.com/a2way-com/laravel-tenant-migrate/pull/12#discussion_r262547444>:

>       {

                $connectionName = $this->argument('connection-name');

                $databaseName = $this->argument('database-name');

-

+               \DB::purge($connectionName);



@crissi<https://github.com/crissi> : Why was this necessary? 🤔

**Comment by crissi** [comment]
It is when one of the commands run in a loop over multiple databases and connections. On its own it works fine

**Comment by budhajeewa** [comment]
@crissi : I tested your changes today, and it worked as expected with Laravel 5.8.

Can you please squash your two commits into one, with the commit message "Upgrade to laravel 5.5 and beyond." and push again?

**Comment by budhajeewa** [comment]
@crissi : Can you please target the pull request at the `dev` branch, and not `master`?

**Comment by budhajeewa** [comment]
@crissi : Please check https://github.com/a2way-com/laravel-tenant-migrate/blob/8d43ed2684142a734c146f73226def17b47b8ee1/CONTRIBUTING.md .
